src/main/java/frc/robot/Constants.java

```diff
@@ -76,4 +76,47 @@ public static class Intake {
     /** Speed we want to run the Intake at. */
     public static final double INTAKE_MOTOR_SPEED = 0.5;
   }
+
+  /** Holds contstants for the Limelights. */
+  public static class Limelight {
+    /** Constants for aiming Limelight. */
+    public static class AimingLimelight {
```

Comment:
Do we need an additional block for aiming the limelight? If so, I would advise moving LIMELIGHT_NAME down a block. If not, I'd remove it.